src/main/kotlin/no/nav/syfo/behandler/api/person/PersonOppfolgingsplanApi.kt

```diff
@@ -21,10 +18,8 @@ private val log: Logger = LoggerFactory.getLogger("no.nav.syfo")
 const val personApiOppfolgingsplanPath = "/api/person/v1/oppfolgingsplan"
 
 fun Route.registerPersonOppfolgingsplanApi(
```

Comment:
Ikke nødvendigvis i denne PR-en, men vi kan vel fjerne Person herfra også?

Reply:
Her er det "Person" som i at det er personen selv som sender inn oppfølgingsplan, så synes det kan være sånn siden det også finnes et system-api for det samme. De andre stedene "person" har blitt borte er det fordi klassen (eller hva det er ) brukes mer generelt nå.